src/docs/node-integration.md

```diff
@@ -26,7 +26,7 @@ Remember to run gclient hooks before compiling.
 
 ```bash
 gclient runhooks
-JOBS=4 make test
+JOBS=72 make test
```

Comment:
Most users probably don't have 72 cores - should we revert this bit?

Reply:
Good point, 
cChanged to `nproc` to be more inclusive.